engines/tsage/staticres.cpp

```diff
@@ -71,7 +71,7 @@ char const *const DEFAULT_SCENE_HOTSPOT = "That accomplishes nothing.";
 char const *const SAVE_ERROR_MSG = "Error occurred saving game. Please do not try to restore this game!";
 char const *const SAVING_NOT_ALLOWED_MSG = "Saving is not allowed at this time.";
 char const *const RESTORING_NOT_ALLOWED_MSG = "Restoring is not allowed at this time.";
-char const *const INV_EMPTY_MSG = "You have nothing in your possesion.";
+char const *const INV_EMPTY_MSG = "You have nothing in your possession.";
```

Comment:
This is in-game data. If the original had the spelling error, it should be kept as it was. Can anyone confirm that?

For all intents and purposes, IMHO it would be preferable to just correct comments, and leave data and code as it is. (And maybe alert a developer about those, so that we can verify those without the risk of breaking code).

Reply:
Well, we do fix typos in some games, e.g. the "I am Choas" line in Loom (which is supposed to be Chaos), plus others

Reply:
e.g. there are typo fixes in the SCUMM engine:
https://wiki.scummvm.org/index.php?title=SCUMM/Game_Enhancements

Reply:
@bluegr : Fixes where the number of characters are identical i.e. Choas -> Chaos are much less problematic than removing/adding ones though i.e. buffer sizes, screen width / line breaking etc.